plugin/src/main/java/io/jenkins/plugins/analysis/core/steps/PublishIssuesStep.java

```diff
@@ -58,6 +58,7 @@
     private String referenceJobName = StringUtils.EMPTY;
     private String referenceBuildId = StringUtils.EMPTY;
     private boolean failOnError = false; // by default, it should not fail on error
+    private boolean isChecksPublishingDisabled = false; // by default, warnings should be published to SCM platforms
```

Comment:
Booleans should not include is at the start the getter will include that

Reply:
Just follows the similar options: https://github.com/jenkinsci/warnings-ng-plugin/blob/b2b19b29067e381c3e4902030e1e6123da8ee1c1/plugin/src/main/java/io/jenkins/plugins/analysis/core/steps/IssuesRecorder.java#L104

Is this style used because the optional is used for disabling a feature? @uhafner